src/components/scenes/CreateWalletSelectCryptoScene.tsx

```diff
@@ -292,4 +298,20 @@ const getStyles = cacheStyles((theme: Theme) => ({
   }
 }))
 
+const CreateWalletSelectCryptoComponent = (props: EdgeSceneProps<'createWalletSelectCrypto'>) => {
+  const { navigation, route } = props
+  const { defaultSelection } = route.params
+
+  return <CreateWalletSelectCrypto navigation={navigation} defaultSelection={defaultSelection} />
+}
+
 export const CreateWalletSelectCryptoScene = React.memo(CreateWalletSelectCryptoComponent)
+
+const CreateWalletSelectCryptoNewAccountComponent = (props: EdgeSceneProps<'createWalletSelectCryptoNewAccount'>) => {
+  const { navigation, route } = props
+  const { defaultSelection, newAccountFlow } = route.params
+
+  return <CreateWalletSelectCrypto navigation={navigation} newAccountFlow={newAccountFlow} defaultSelection={defaultSelection} />
+}
+
+export const CreateWalletSelectCryptoNewAccountScene = React.memo(CreateWalletSelectCryptoNewAccountComponent)
```

Comment:
What is the point of this change? What was wrong with the way it was before; `newAccountFlow` param optional for `createWalletSelectCrypto` scene?

Reply:
I see the changes are in https://github.com/EdgeApp/edge-react-gui/pull/4384. So this will be fixed when rebased onto master.